docs/source/conf.py

```diff
@@ -170,7 +170,7 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
-html_css_files = ['link-box.css']
+html_css_files = ['link-box.css', 'custom.css']
```

Comment:
If you remove this change, it should resolve your merge conflict. Then move the `custom.css` content to `docs/source/_static/aiida-custom.css` that will now be present

Reply:
Great, thanks for figuring that out for me, @chrisjsewell! 😄